base_classes/NXlog.nxdl.xml

```diff
@@ -22,57 +22,57 @@
 # For further information, see http://www.nexusformat.org
 -->
 <definition
-	name="NXlog" 
-	version="1.1" 
-    type="group" 
+	name="NXlog"
+	version="1.1"
+    type="group"
     extends="NXobject"
 	category="base"
-	xmlns="http://definition.nexusformat.org/nxdl/3.1" 
+	xmlns="http://definition.nexusformat.org/nxdl/3.1"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
     >
     <doc>
 		Information recorded as a function of time.
-		
+
 		Description of information that is recorded against
 		time. There are two common use cases for this:
 
 		- When logging data such as temperature during a run
 		- When data is taken in streaming mode data acquisition,
 		  i.e. just timestamp, value pairs are stored and
-		  correlated later in data reduction with other data, 
+		  correlated later in data reduction with other data,
 
-		
-		It both cases NXlog contains 
+
+		It both cases NXlog contains
```

Comment:
remove "It", replace with "In"

Reply:
True, we should fix all we can. I didn't read that part... ;)